ffv1.md

```diff
@@ -279,24 +279,32 @@ For each plane of each slice, a Quantization Table Set is selected from an index
 
 Background: in first implementations of FFV1 bitstream, the index for Cb and Cr planes was stored even if it is not used (chroma_planes set to 0), this index is kept for version <= 3 in order to keep compatibility with FFV1 bitstreams in the wild.
 
-## Color space
+## Components
 
-FFV1 supports two color spaces: YCbCr and JPEG2000-RCT. Both color spaces allow an optional Alpha plane that can be used to code transparency data.
+FFV1 supports 1 Luma component, optionaly 2 Chroma components and 1 Alpha component that can be used to code transparency data.
```

Comment:
suggest `1 Luma component, optionally 2 Chroma components, and optionally 1 Alpha component`

& optionaly was a typo